align/pdk/finfet/transistor_array.py

```diff
@@ -324,9 +324,9 @@ def _find_y_bounds(pin, wire):
                 self.drop_via(self.v2)
 
         # Expose pins
-        for term in self.terminals:
-            if term['netName'] is not None and term['layer'] in ['M2', 'M3']:
-                term['pin'] = term['netName']
+        #for term in self.terminals:
+        #    if term['netName'] is not None and term['layer'] in ['M2', 'M3']:
+        #        term['pin'] = term['netName']
```

Comment:
Instead of commenting out, we could change the logic by setting `term['netType']='pin'` ?